### PR TITLE
Bv 57/refactor line chart

### PR DESCRIPTION
**Pull request by wojciech-nowakowski**

### CODE REVIEW

File: src/components/LineChart/LineChart-container.tsx [hunk 1]
```diff
@@ -1,216 +1,85 @@
...
-          onClick={(): void => {
-            setTimeout(() => submitChart(), 100);
-          }}
-          style={{ marginLeft: 20, marginTop: 10 }}
-        >
-          Submit
-        </Button>
-      </div>
-      <h1 style={{ textAlign: 'center' }}>{config.title}</h1>
+    <div style={{ height: '400px', background: 'linear-gradient(193.66deg, rgba(37, 45, 72, 0) 49.65%, #252D48 100%)', padding: 10, borderRadius: 5 }}>
```
Comment (dariuszsobkowicz) [inline src/components/LineChart/LineChart-container.tsx]: isn't it better to convert the style to class because values are not dynamic?

**Review by dariuszsobkowicz — APPROVED** [review]

File: src/components/LineChart/Line-view.test.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import LineChart from './Line';
+import LineChart from './Line-view';
```
Comment (damianlewandowski) [inline src/components/LineChart/Line-view.test.tsx]: Confusing, should be `LineView` instead or change the filename to `LineChart-view`

File: src/components/LineChart/LineChart-container.tsx [hunk 3]
```diff
@@ -1,216 +1,86 @@
...
 import {
-  convertTimeStamp,
   convertTimeStampToHours,
-  getSelectedDate
 } from './helpers';
 
-const LineCharts = (props:any): React.ReactElement => {
-  const {blokchain, actions} = props;
+import testData from './data';
+import classes from './LineChart.module.css';
```
Comment (damianlewandowski) [inline src/components/LineChart/LineChart-container.tsx]: we are not using CSS in this project, use JSS instead

File: src/components/LineChart/LineChart-container.tsx [hunk 4]
```diff
@@ -1,216 +1,86 @@
...
+
+const LineCharts = (props: any): React.ReactElement => {
+  const { blokchain, actions } = props;
 
-  const [dateFrom, setDateFrom] = useState(getSelectedDate(7));
   const [label, setLabel] = useState([
```
Comment (damianlewandowski) [inline src/components/LineChart/LineChart-container.tsx]: This is an array, use plural form

File: src/components/LineChart/LineChart-container.tsx [hunk 5]
```diff
@@ -1,216 +1,86 @@
...
-    setTimeout(() => setBuyer(e.target.value), 100);
-  };
 
-  const handleSellerChange = (e: any) => {
-    setTimeout(() => setSeller(e.target.value), 100);
-  };
-
-  const filterChart = (blokchain: Block[], chartType: string): void => {
+  const filterChart = (): void => {
     const labels: any[] = [];
```
Comment (damianlewandowski) [inline src/components/LineChart/LineChart-container.tsx]: is `:any[]` necessary here?

File: src/components/LineChart/LineChart.module.css [hunk 6]
```diff
@@ -0,0 +1,6 @@
+.LineChartContainer {
+    height: 400px;
+    background: linear-gradient(193.66deg, rgba(37, 45, 72, 0) 49.65%, #252D48 100%);
+    padding: 10px;
+    border-radius: 5px;
+}
```
Comment (damianlewandowski) [inline src/components/LineChart/LineChart.module.css]: Use JSS

**Review by damianlewandowski — CHANGES_REQUESTED** [review]

**Review by damianlewandowski — APPROVED** [review]

**Pull request merged**